mdm/common/src/main/java/com/gentics/mesh/core/image/ImageManipulator.java

```diff
@@ -47,38 +47,36 @@
     Single<File> handleS3Resize(String bucketName, String s3ObjectKey, String filename, ImageManipulationParameters parameters);
```

Comment:
Here it looks like only the indentation changed, so at least this file can be reverted.

Reply:
UPD: caught the old commit, all ok in the end.